app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=("announcement #{ 'future' if announcement.valid_from > DateTime.now }")
+  h3
+    = announcement.title
+
+  - if can? :manage, Course::Announcement
+    .pull-right
```

Comment:
@weiqingtoh Thanks !  if all you guys prefer <code>div</code>, then I'll change it. 

@lowjoel is right, verbosity vs terseness